sixtyfps_runtime/corelib/graphics/image.rs

```diff
@@ -304,6 +304,14 @@ impl Image {
             ImageInner::EmbeddedImage(buffer) => [buffer.width() as _, buffer.height() as _].into(),
         }
     }
+
+    /// Returns the path of the image on disk, if it was constructed via [`Self::load_from_path`].
+    pub fn path(&self) -> Option<&std::path::Path> {
```

Comment:
(this function is not tested, doc tests would be nice, if possible)

Reply:
Ah yes, it's only implicit through the C++ test. I'd prefer an explicit API test though, as otherwise I'm not sure what to use as value of the path. It feels strange if the documentation example uses `Path::new(env!("CARGO_MANIFEST_DIR")).join("../../tests/cases/blah.png")`. Unless that's okay perhaps?

Reply:
Ah, doesn't look too bad after adding it :-)